work/fifth/index.html

```diff
@@ -99,6 +99,25 @@
       // .change() セレクトボックスの値の変更をキャッチする関数
       // $().each(function(index, element){}) DOM要素を1つずつループ処理をする。
       // eachには引数を二つ持たせることができ、第一引数はDOM要素のインデックス番号、第二引数ではDOM要素自体が取得できます
+    $(function(){
+      $('select').click(filter);
```

Comment:
[imo]
今回はフィルタリング実装になるので、`.change` での実装推奨です！

```suggestion
      $('.select').change(function () {
          //  コード
      });
```